WordPress/build.gradle

```diff
@@ -125,6 +125,7 @@ android {
         buildConfigField "boolean", "JETPACK_BLOGGING_REMINDERS_SYNC", "false"
         buildConfigField "boolean", "JETPACK_READER_SAVED_POSTS", "false"
         buildConfigField "boolean", "JETPACK_PROVIDER_SYNC", "false"
+        buildConfigField "boolean", "JETPACK_MIGRATION_FLOW", "false"
```

Comment:
I guess this is only showing in the diff because this PR targets `trunk` instead of the PR that implements the flag as a base?

Reply:
Indeed, this is the same for the next PRs for the Welcome Screen in the migration flow, and it's probably not the best approach. I hope we can discuss something today or tomorrow to enable us to better progress and review these 🙇